portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/BlogsEvents.testcase

```diff
@@ -146,7 +142,7 @@ definition {
 			task ("Start Har recording") {
 				ProxyUtil.startHarRecording("blogClicked");
 
-				Pause(locator1 = "5000");
+				ACUtils.waitForSendingRequest();
```

Comment:
Hey @calvinctvo, I think you could put this macro for after the iteration in DXP. I think you could put. What do you think?

```
task ("Wait for sending requests") {
	ACUtils.waitForSendingRequest();
}
```

After

```
Click(
	key_entryTitle = "Blogs AC Title",
	locator1 = "BlogsEntry#TITLE");
```